app/services/async_plan_provisioner.rb

```diff
@@ -1,4 +1,4 @@
-class PlanProvisioner
+class AsyncPlanProvisioner
```

Comment:
I'd say that the sync/async nature of this service class is generally not relevant but I can see how this rename would help external people understand what it does.

Reply:
Agreed, I renamed this only to improve external clarity.